evergreen/evergreen.yml

```diff
@@ -224,6 +224,7 @@ functions:
 
   pack-packages:
     - command: shell.exec
+      type: test
```

Comment:
This will mark failed script as "Failed" (red) instead of "System Failure" (purple)